src/views/Movies-Components/Home.js

```diff
@@ -1,18 +1,37 @@
 import { Grid } from "@mui/material";
 import React from "react";
+import { TopRatedMovies } from "./TopRatedMovies";
+import { Navigation } from "swiper";
+import "swiper/css";
+import "swiper/css/navigation";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { first_path } from "./Constants";
 
+import { useGetPopularMoviesQuery } from "../../services/MovieServices";
 export const Home = () => {
+  const { data } = useGetPopularMoviesQuery();

```

Comment:
I think we should always have some sort of default value for data as it may be undefined in the beginning. Something like this could help:
`const { data = []} = useQuery()`